sig-docs/README.md

```diff
@@ -99,9 +99,15 @@ The following [subprojects][subproject-definition] are owned by sig-docs:
 ### reference-docs
 - **Owners:**
   - [kubernetes-sigs/reference-docs](https://github.com/kubernetes-sigs/reference-docs/blob/master/OWNERS)
+### subproject-definition
+- **Owners:**
+  - [kubernetes/community/governance.md#subprojects](https://github.com/kubernetes/community/blob/master/governance.md#subprojects)
 ### website
 - **Owners:**
   - [kubernetes/website](https://github.com/kubernetes/website/blob/master/OWNERS)
+### working-group-definition
+- **Owners:**
+  - [kubernetes/community/governance.md#working-groups](https://github.com/kubernetes/community/blob/master/governance.md#working-groups)
```

Comment:
I think this place is better for the link to the WG definition if it's not there
https://github.com/kubernetes/community/blob/master/governance.md#working-groups

This area is for named SIG Docs subprojects